src/ui/component/claimPreview/view.jsx

```diff
@@ -219,7 +232,7 @@ const ClaimPreview = forwardRef<any, {}>((props: Props, ref: any) => {
               </div>
             )}
           </div>
-          <ClaimTags uri={uri} type={type} />
+          {properties !== undefined ? properties : <ClaimTags uri={uri} type={type} />}
```

Comment:
What should properties be when you want the positive action to happen? I suspect this check should be stronger

Reply:
Mainly `properties={false}` to hide them.